### PR TITLE
docs: add dune dashboard link

### LINKED ISSUE #965: Analytics dashboard
It would be useful create a [dune](https://dune.com/) analytics dashboard for the Ubiquity protocol ([example](https://dune.com/KARTOD/AAVE-Mega-Dashboard)).

What data to display in the dashboard:
- [UbiquityDollarToken](https://etherscan.io/address/0xb6919Ef2ee4aFC163BC954C5678e2BB570c2D103): total supply
- [UBQ](https://etherscan.io/address/0x4e38D89362f7e5db0096CE44ebD021c3962aA9a0): total supply
- [UbiquityPoolFacet](https://etherscan.io/address/0xED3084c98148e2528DaDCB53C56352e549C488fA) (diamond proxy link): LUSD collateral [information](https://github.com/ubiquity/ubiquity-dollar/blob/c5891e9c06148dbbfdf7b8e410381c99c0b55ad9/packages/contracts/src/dollar/libraries/LibUbiquityPool.sol#L265)
- [UbiquityPoolFacet](https://etherscan.io/address/0xED3084c98148e2528DaDCB53C56352e549C488fA): current collateral [ratio](https://github.com/ubiquity/ubiquity-dollar/blob/c5891e9c06148dbbfdf7b8e410381c99c0b55ad9/packages/contracts/src/dollar/libraries/LibUbiquityPool.sol#L302)
- [UbiquityPoolFacet](https://etherscan.io/address/0xED3084c98148e2528DaDCB53C56352e549C488fA): collateral [value](https://github.com/ubiquity/ubiquity-dollar/blob/c5891e9c06148dbbfdf7b8e410381c99c0b55ad9/packages/contracts/src/dollar/libraries/LibUbiquityPool.sol#L311) in USD
- [UbiquityPoolFacet](https://etherscan.io/address/0xED3084c98148e2528DaDCB53C56352e549C488fA): free collateral [balance](https://github.com/ubiquity/ubiquity-dollar/blob/c5891e9c06148dbbfdf7b8e410381c99c0b55ad9/packages/contracts/src/dollar/libraries/LibUbiquityPool.sol#L348) that can be borrowed by AMO minters
- [UbiquityPoolFacet](https://etherscan.io/address/0xED3084c98148e2528DaDCB53C56352e549C488fA): Dollar [price](https://github.com/ubiquity/ubiquity-dollar/blob/c5891e9c06148dbbfdf7b8e410381c99c0b55ad9/packages/contracts/src/dollar/libraries/LibUbiquityPool.sol#L384) in USD
- [UbiquityPoolFacet](https://etherscan.io/address/0xED3084c98148e2528DaDCB53C56352e549C488fA): UBQ [price](https://github.com/ubiquity/ubiquity-dollar/blob/c5891e9c06148dbbfdf7b8e410381c99c0b55ad9/packages/contracts/src/dollar/libraries/LibUbiquityPool.sol#L433) in USD
- [UbiquityPoolFacet](https://etherscan.io/address/0xED3084c98148e2528DaDCB53C56352e549C488fA): total amount of LUSD collateral in the pool

Smart contract addresses: https://github.com/ubiquity/ubiquity-dollar/wiki/24.-Smart-Contracts

### PR DESCRIPTION
**Pull request by YanVictorSN**
Resolves #965

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -160,6 +160,10 @@ If successful it will output the accounts generated from the test mnemonic (`tes
 
 We have a [Wiki!](https://github.com/ubiquity/ubiquity-dollar/wiki) feel free to browse it as there is a lot of useful information about the whole repo
 
+## Analytics Dashboard
+
+We have created a [Dune Analytics](https://dune.com/yanvictoreth/ubiquity-protocol) dashboard to visualize data for Ubiquity Protocol.
```
Comment (0x4007) [inline README.md]: Looks like you mixed up our governance token with our dollar token on the dashboard regarding circulating supply!

Comment (YanVictorSN) [reply README.md]: Yes. It's fixed now. Thanks!

**Comment by rndquu** [comment]
@YanVictorSN Hey, a few questions:
1. Is it possible to transfer the dashboard ownership to the [ubiquity_dao](https://dune.com/ubiquity_dao) team?
2. How often is the dashboard updated? I mean do we need to update it manually or dune automatically updates it once in a while?
3. The [lusd-collateral-information](https://dune.com/queries/4114685/6935411) table in dashboard is missing [these](https://github.com/ubiquity/ubiquity-dollar/blob/c5891e9c06148dbbfdf7b8e410381c99c0b55ad9/packages/contracts/src/dollar/libraries/LibUbiquityPool.sol#L116-L131) properties from [this](https://github.com/ubiquity/ubiquity-dollar/blob/c5891e9c06148dbbfdf7b8e410381c99c0b55ad9/packages/contracts/src/dollar/libraries/LibUbiquityPool.sol#L265) method. Is it possible to display all of them?
4. Some of the queries are private (like [total_amount_lusd_collateral_pool](https://dune.com/queries/4115231/6935398), I'm getting the `404 Page Not Found or Private` error). Could you make all of the queries public? I want to understand how everything is calculated.

**Comment by 0x4007** [comment]
> @YanVictorSN Hey, a few questions:
> 
> 1. Is it possible to transfer the dashboard ownership to the [ubiquity_os](https://dune.com/ubiquity_os) team?

I prefer https://dune.com/ubiquity_dao @rndquu I made [you admin](https://dune.com/t/ubiquity_dao/rQanZbQWvxv5)

**Comment by YanVictorSN** [comment]
> @YanVictorSN Hey, a few questions:
> 
> 1. Is it possible to transfer the dashboard ownership to the [ubiquity_dao](https://dune.com/ubiquity_dao) team?
> 2. How often is the dashboard updated? I mean do we need to update it manually or dune automatically updates it once in a while?
> 3. The [lusd-collateral-information](https://dune.com/queries/4114685/6935411) table in dashboard is missing [these](https://github.com/ubiquity/ubiquity-dollar/blob/c5891e9c06148dbbfdf7b8e410381c99c0b55ad9/packages/contracts/src/dollar/libraries/LibUbiquityPool.sol#L116-L131) properties from [this](https://github.com/ubiquity/ubiquity-dollar/blob/c5891e9c06148dbbfdf7b8e410381c99c0b55ad9/packages/contracts/src/dollar/libraries/LibUbiquityPool.sol#L265) method. Is it possible to display all of them?
> 4. Some of the queries are private (like [total_amount_lusd_collateral_pool](https://dune.com/queries/4115231/6935398), I'm getting the `404 Page Not Found or Private` error). Could you make all of the queries public? I want to understand how everything is calculated.

1. Yes, it's possible. We need to create a team on Dune and then I can transfer the dashboard and queries. 
2. It's possible to schedule them to run automatically (by minute, hour, day, or week). Running all queries right now would cost us 90 credits and we have 2,700 available per month.
3. I see. I think it's possible if I change how I call the query. Dune can't directly access functions or variables only data that triggers an on-chain action like an event. But we can use an API to call the function internally. I'll work on that.
4. Sure. I didn't want to expose the API key so I made it private. I'll remove it now.

**Comment by YanVictorSN** [comment]
> > @YanVictorSN Hey, a few questions:
> > 
> > 1. Is it possible to transfer the dashboard ownership to the [ubiquity_os](https://dune.com/ubiquity_os) team?
> 
> I prefer https://dune.com/ubiquity_dao @rndquu I made [you admin](https://dune.com/t/ubiquity_dao/rQanZbQWvxv5)

@0x4007 can you add me to the Ubiquity DAO team so I can share the dune dashboard? @yanvictoreth

**Comment by YanVictorSN** [comment]
> 3\. The [lusd-collateral-information](https://dune.com/queries/4114685/6935411) table in dashboard is missing [these](https://github.com/ubiquity/ubiquity-dollar/blob/c5891e9c06148dbbfdf7b8e410381c99c0b55ad9/packages/contracts/src/dollar/libraries/LibUbiquityPool.sol#L116-L131) properties from [this](https://github.com/ubiquity/ubiquity-dollar/blob/c5891e9c06148dbbfdf7b8e410381c99c0b55ad9/packages/contracts/src/dollar/libraries/LibUbiquityPool.sol#L265) method. Is it possible to display all of them?

@rndquu 

Manage to get 3 more infos:  
1- collateralAddress
2- poolStorage.collateralPriceFeedAddresses
3- poolStorage.collateralPriceFeedStalenessThresholds

Working on the others.

**Comment by rndquu** [comment]
> > > @YanVictorSN Hey, a few questions:
> > > 
> > > 1. Is it possible to transfer the dashboard ownership to the [ubiquity_os](https://dune.com/ubiquity_os) team?
> > 
> > 
> > I prefer https://dune.com/ubiquity_dao @rndquu I made [you admin](https://dune.com/t/ubiquity_dao/rQanZbQWvxv5)
> 
> @0x4007 can you add me to the Ubiquity DAO team so I can share the dune dashboard? @yanvictoreth

Done, I suppose the "Editor" role has enough permissions to transfer the dashboard

**Comment by YanVictorSN** [comment]
> > > > @YanVictorSN Hey, a few questions:
> > > > 
> > > > 1. Is it possible to transfer the dashboard ownership to the [ubiquity_os](https://dune.com/ubiquity_os) team?
> > > 
> > > 
> > > I prefer https://dune.com/ubiquity_dao @rndquu I made [you admin](https://dune.com/t/ubiquity_dao/rQanZbQWvxv5)
> > 
> > 
> > @0x4007 can you add me to the Ubiquity DAO team so I can share the dune dashboard? @yanvictoreth
> 
> Done, I suppose the "Editor" role has enough permissions to transfer the dashboard

It worked. I just transferred the ownership. Thanks.

**Comment by rndquu** [comment]
> > > > > @YanVictorSN Hey, a few questions:
> > > > > 
> > > > > 1. Is it possible to transfer the dashboard ownership to the [ubiquity_os](https://dune.com/ubiquity_os) team?
> > > > 
> > > > 
> > > > I prefer https://dune.com/ubiquity_dao @rndquu I made [you admin](https://dune.com/t/ubiquity_dao/rQanZbQWvxv5)
> > > 
> > > 
> > > @0x4007 can you add me to the Ubiquity DAO team so I can share the dune dashboard? @yanvictoreth
> > 
> > 
> > Done, I suppose the "Editor" role has enough permissions to transfer the dashboard
> 
> It worked. I just transferred the ownership. Thanks.

Pls mark the PR as "ready for review" once you're done with the dashboard

**Comment by YanVictorSN** [comment]
> 3. The [lusd-collateral-information](https://dune.com/queries/4114685/6935411) table in dashboard is missing [these](https://github.com/ubiquity/ubiquity-dollar/blob/c5891e9c06148dbbfdf7b8e410381c99c0b55ad9/packages/contracts/src/dollar/libraries/LibUbiquityPool.sol#L116-L131) properties from [this](https://github.com/ubiquity/ubiquity-dollar/blob/c5891e9c06148dbbfdf7b8e410381c99c0b55ad9/packages/contracts/src/dollar/libraries/LibUbiquityPool.sol#L265) method. Is it possible to display all of them?

@rndquu It's done. We have displayed all the lusd colleteral information.

**Comment by YanVictorSN** [comment]
hi @rndquu  any update on this task?

**Comment by 0x4007** [comment]
@rndquu

**Comment by YanVictorSN** [comment]
Hi @rndquu can you assign me on the issue?

**Comment by rndquu** [comment]
@YanVictorSN Could you merge the latest `development` branch?

**Comment by YanVictorSN** [comment]
> @YanVictorSN Could you merge the latest `development` branch?

Sure.

**Comment by YanVictorSN** [comment]
> @YanVictorSN Could you merge the latest `development` branch?

@rndquu I think it's done.

**Review by rndquu — APPROVED** [review]
@YanVictorSN Sorry for a long review, the PR looks good

**Pull request merged**